README.md

```diff
@@ -488,7 +488,8 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [spider-gazelle](https://github.com/spider-gazelle/spider-gazelle) - A Rails esque web framework with a focus on speed and extensibility
 
 # Community
- * [Chicago Crystal](https://podcast.chicagocrystal.org) - A podcast about the Crystal language
+ * [Chicago Crystal Podcast](https://podcast.chicagocrystal.org)
```

Comment:
Thanks! This link gives an error

![image](https://user-images.githubusercontent.com/3624712/87969317-9168f200-caca-11ea-8924-4dcb1ea33463.png)

Reply:
we do need to add ssl.  I am not sure how to do this yet since it is a redirect to buzzsprout.